react-app/src/components/UI/GridButton.tsx

```diff
@@ -0,0 +1,15 @@
+export default function GridConfigButton({
+  name,
+  handleClick,
+  style,
+}: {
+  name?: string;
```

Comment:
why the name is optional here? Every button should have some caption

Reply:
who knows...